docs/CHANGELOG.md

```diff
@@ -1,7 +1,33 @@
 ### vNEXT 
+
+**Breaking changes:**
+
+It is no longer guaranteed that a field's value is available under `source[fieldName]` in a custom resolver. Instead, custom resolvers on non-trivial fields need to use GraphQL's default resolver to get the field value:
```

Comment:
can we change "non-trivial" for "fields with multiple aliases"?

Reply:
Not sure - whether or not there are multiple aliases depends on the client query, so the schema should probably assume this might always be the case.

What I wanted to express with "non-trivial" is "types of fields that handle multiple aliases this way because they might return different data for each of them" 😅

Reply:
Ah ok then link to the definition of non trivial so people reading the changelog know what it means 😀

Reply:
Done